pkg/version/version.go

```diff
@@ -40,6 +43,11 @@ func GetVersion() string {
 	return version
 }
 
+// GetGitCommitId returns the git commit id from which it is being built
+func GetGitCommitId() string {
```

Comment:
This should be `GetGitCommitID` 

I'd normally say this should be just `GitCommitID()`, but we should be consistent with the other existing function calls.

Reply:
I would also rather GitCommitID even if it is not consistent with others, over time we can improve others